app/src/sharedTest/java/org/oppia/app/parser/HtmlParserTest.kt

```diff
@@ -0,0 +1,84 @@
+package org.oppia.app.parser
+
+import android.app.Activity
+import android.content.Intent
+import android.content.pm.ActivityInfo
+import android.text.Spannable
+import android.widget.TextView
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import com.google.common.truth.Truth.assertThat
+import org.hamcrest.Matchers.not
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.testing.HtmlParserTestActivity
+import org.oppia.util.parser.HtmlParser
+import javax.inject.Inject
+
+/** Tests for [HtmlParser]. */
+@RunWith(AndroidJUnit4::class)
+class HtmlParserTest {
+
+  private lateinit var launchedActivity: Activity
+  @Inject
+  lateinit var htmlParserFactory : HtmlParser.Factory
+
+  @get:Rule
+  var activityTestRule: ActivityTestRule<HtmlParserTestActivity> = ActivityTestRule(
+    HtmlParserTestActivity::class.java, /* initialTouchMode= */ true, /* launchActivity= */ false
+  )
+
+  @Before
+  fun setUp() {
+    Intents.init()
+    val intent = Intent(Intent.ACTION_PICK)
+    launchedActivity = activityTestRule.launchActivity(intent)
+  }
+
+  @Test
+  fun testHtmlContent_handleCustomOppiaTags_parsedHtmldisplaysStyledText() {
+    activityTestRule.activity.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
+    activityTestRule.activity.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
```

Comment:
Please remove these two lines--as we discussed via chat, they aren't actually affecting the behavior of the test. Ditto below.

Reply:
removed orientation lines.